lib/Bindings/Python/OMModule.cpp

```diff
@@ -445,13 +445,36 @@ static MlirAttribute omPythonValueToPrimitive(PythonPrimitive value,
   if (auto *attr = std::get_if<nb::str>(&value)) {
     auto str = nb::cast<std::string>(*attr);
     auto strRef = mlirStringRefCreate(str.data(), str.length());
-    return mlirStringAttrGet(ctx, strRef);
+    auto omStringType = omStringTypeGet(ctx);
+    return mlirStringAttrTypedGet(omStringType, strRef);
   }
 
   if (auto *attr = std::get_if<nb::bool_>(&value)) {
     return mlirBoolAttrGet(ctx, nb::cast<bool>(*attr));
   }
 
+  // For a python list try constructing OM list attribute. The element
+  // type must be uniform.
+  if (auto *attr = std::get_if<nb::list>(&value)) {
+    if (attr->size() == 0)
+      throw nb::type_error("Empty list is prohibited now");
```

Comment:
This restriction is because we need at least one element to infer the result type, right?

Reply:
Yes. We could use !om.any as an element type (or maybe just using om.any regardless of element type might be better) but I think we can modify it later.